javascripts/discourse/initializers/dc-header.js.es6

```diff
@@ -23,9 +22,8 @@ export default {
         },
         didInsertElement() {
           this._super();
-          // TODO: whenever the dc-dropdown-component is available use that instead
-          loadScript(
-            "https://unpkg.com/@debtcollective/dc-header-component@1.5.0/dist/header/header.js"
+          loadStencilScript(
+            "https://unpkg.com/@debtcollective/dc-dropdown-component@1.6.3/dist/dropdown-component/dropdown-component.esm.js"
```

Comment:
@duranmla I think the nomodule version needs to point to another script (the cjs version instead of esm). I'm not 100% sure though.

Reply:
This has been tested already and it is working fine by now. But thanks to jump in! ❤️